docs/src/diagnostics/make_diagnostic_table.jl

```diff
@@ -0,0 +1,27 @@
+import ClimaLand as CL
+using PrettyTables
+
+# Print all available diagnostics to an ASCII table
+
```

Comment:
```suggestion
CL.define_diagnostics!(nothing)
```

Reply:
I added that, but while this works, we can't use `get_diagnostic_variable` when just using ClimaLand.Diagnostics since we don't call `define_diagnostics` in src